.evergreen/config_generator/components/loadbalanced.py

```diff
@@ -0,0 +1,106 @@
+from shrub.v3.evg_build_variant import BuildVariant
+from shrub.v3.evg_command import EvgCommandType, FunctionCall, expansions_update
+from shrub.v3.evg_task import EvgTaskRef, EvgTaskDependency
+
+from config_generator.components.funcs.bootstrap_mongo_orchestration import BootstrapMongoOrchestration
+from config_generator.components.funcs.fetch_det import FetchDET
+from config_generator.components.funcs.upload_build import UploadBuild
+from config_generator.components.funcs.run_simple_http_server import RunSimpleHTTPServer
+from config_generator.components.funcs.fetch_build import FetchBuild
+from config_generator.components.funcs.run_tests import RunTests
+from config_generator.etc.utils import Task, bash_exec
+
+
+def functions():
+    return {
+        'start load balancer': [
```

Comment:
```suggestion
        'start-load-balancer': [
```
Recommend using `-` instead of spaces for consistency in new EVG config generator scripts.